src/presentational-components/shared/loader-placeholders.js

```diff
@@ -215,3 +216,9 @@ ListLoader.defaultProps = {
 };
 
 export const OrderDetailToolbarPlaceholder = () => <Skeleton height={70} />;
+
+export const PlatformToolbarPlaceholder = () => (
+  <StyledToolbar className="pf-u-pr-lg pf-u-pl-lg pf-u-pb-md pf-u-pt-md">
```

Comment:
"pf-u-pr-lg pf-u-pl-lg pf-u-pb-md pf-u-pt-md"

 the first two could be collapsed into one

"pf-u-p-lg pf-u-pb-md pf-u-pt-md"

but not sure if the PF css allows this.

Reply:
Hey, I am happy that it even works :smile: who knows what it will do when we try to be clever.